python/cli/prelude_cli/views/partner.py

```diff
@@ -14,52 +14,52 @@ def partner(ctx):
 
 
 @partner.command('attach')
-@click.argument('name')
+@click.argument('partner-code')
```

Comment:
two questions:
1. is partner-code the best name?
2. how could a user become aware of the options?

Reply:
I'll add help text and elaborate. Might be a good time to throw in the enum as well.